pages/org-col-info.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>USC-CITE | Organization Information</title>
+
+    <!-- Box Icon Styles -->
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="styles/global.css">
+    <link rel="stylesheet" href="styleS/org-uni-info.css">
+</head>
+<body>
+    <header>
+        <div class="header__container">
+            <div class="header__banner">
+                <a href="/"><img src="/pages/assets/logos/USC-logo.png" alt="USC-logo" class="header__logo" />
+                    <div class="header__text">
+                        <h1>West Visayas State University</h1>
+                        <h2>University Student Council</h2>
+                    </div>
+                </a>
+            </div>
+            <div class="header__btns">
+                <div class="header__menu">
+                    <a href="/browse" data-js="menu_btn">
+                        <p>Menu</p>
+                        <img alt="" src="/pages/assets/icons/chevron_down.svg" data-js="menu_btn_icon" />
+                    </a>
+                </div>
+                <div class="header__search">
+                    <a href="/search" data-js="search_btn">
+                        <p class="sr_only">Search</p>
+                        <img alt="Search" src="/pages/assets/icons/magnifier.svg" />
+                    </a>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <!-- main section here -->
+    <main class="main">
+        <div class="main__container">
+            <img src="assets/logos/ICON-logo.png" alt="" class="org-logo">
+            <div class="main__organization">
+                <h2 class="org-name">ICON Publication</h2>
+                <p class="org-type">College-Based Organization</p>
+                <p class="org-type">College of CICT</p>
+            </div>
+        </div>
+
+        <div class="main__container second-container">
+            <div class="main__content">
+                <p class="main__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sodales dolor pulvinar magna interdum elementum. Maecenas tortor risus, dapibus at cursus a, suscipit at purus. Morbi id congue lacus, vel tincidunt risus. Nunc vitae blandit turpis. Duis dignissim ut leo et lobortis. Ut ac metus feugiat, fringilla est nec, consectetur.</p>
+                <div class="rectangle"></div>
```

Comment:
You know the `rectangle` class is a placeholder for images right? It's for one cover image but I have plans to test it with multiple images using our CMS. Just take note of it and I might demonstrate how it will work.